# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/wr/webrender/src/shade.rs
# Commit: 15da94c6a7b1
# Full Hash: 15da94c6a7b19f971d3dffaa4cab8edb8e41cb62
# Author: Glenn Watson <git@intuitionlibrary.com>
# Date: 2020-12-15 09:29:54
# Regressor Bug: 1682365
# File Overlap Count: 1
# Description:
#   Bug 1682365 - Pt 2 - Remove texture array usage from render task graph. r=nical
#   
#   With this change, all color/alpha intermediate surfaces are individual
#   textures, rather than texture arrays.
#   
# ==============================================================================

diff -r 72ddac486037 -r 15da94c6a7b1 gfx/wr/webrender/src/shade.rs
--- a/gfx/wr/webrender/src/shade.rs	Tue Dec 15 01:46:51 2020 +0200
+++ b/gfx/wr/webrender/src/shade.rs	Mon Dec 14 21:48:41 2020 +0000
@@ -25,7 +25,7 @@
 pub(crate) fn get_feature_string(kind: ImageBufferKind) -> &'static str {
     match kind {
         ImageBufferKind::Texture2D => "TEXTURE_2D",
-        ImageBufferKind::Texture2DArray => "",
+        ImageBufferKind::Texture2DArray => "TEXTURE_2D_ARRAY",
         ImageBufferKind::TextureRect => "TEXTURE_RECT",
         ImageBufferKind::TextureExternal => "TEXTURE_EXTERNAL",
     }